### PR TITLE
[zh] sync liveness-readiness-startup-probes dynamic-resource-allocation install-kubeadm

### PR DESCRIPTION
**Pull request by asa3311**

### CODE REVIEW

File: content/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 1]
```diff
@@ -133,6 +133,35 @@ documentation for the plugins about what port(s) those need.
...
+  This can be changed by setting a `swapBehavior`, again in the kubelet configuration file. To use swap,
+  set a `swapBehavior` other than the default `NoSwap` setting.
+  See [Swap memory management](/docs/concepts/architecture/nodes/#swap-memory) for more details.
+* To disable swap, `sudo swapoff -a` can be used to disable swapping temporarily.
+  To make this change persistent across reboots, make sure swap is disabled in
+  config files like `/etc/fstab`, `systemd.swap`, depending how it was configured on your system.
+-->
+## 交换分区的配置 {#swap-configuration}
+
+交换分区的配置。kubelet 的默认行为是在节点上检测到交换内存时无法启动。
```
Comment (tengqm) [inline content/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:153]:
```suggestion
kubelet 的默认行为是在节点上检测到交换内存时无法启动。
```

File: content/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 2]
```diff
@@ -133,6 +133,35 @@ documentation for the plugins about what port(s) those need.
...
+这意味着要么禁用交换（swap）功能，要么让 kubelet 容忍交换。
+
+* 若需允许交换分区（swap），请在 kubelet 配置文件中添加 `failSwapOn: false`，或通过命令行参数指定。
+  注意：即使设置了 `failSwapOn: false`，工作负载默认情况下仍无法访问交换空间。
+  可以通过在 kubelet 配置文件中设置 `swapBehavior` 来修改此设置。若要使用交换空间，
+  请将 `swapBehavior` 设置为非默认的 `NoSwap` 值。
```
Comment (mengjiao-liu) [inline content/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:159]: 有点歧义。
```suggestion
  请设置 `swapBehavior` 的值，这个值不能是默认的 `NoSwap`。
```

**Review by mengjiao-liu — COMMENTED** [review]
/approve

**Pull request merged**